realtimenet/controller.py

```diff
@@ -0,0 +1,131 @@
+from typing import List
+from typing import Union
+
+from realtimenet.camera import VideoSource
+from realtimenet.camera import VideoStream
+from realtimenet.display import DisplayResults
+from realtimenet.engine import InferenceEngine
+from realtimenet.downstream_tasks.postprocess import PostProcessor
+
+import cv2
+import numpy as np
+import torch.nn as nn
+
+
+class Controller:
+
+    def __init__(
+            self,
+            neural_network: nn.Module,
+            post_processors: Union[PostProcessor, List[PostProcessor]],
+            results_display: DisplayResults,
+            camera_id: int,
```

Comment:
I think we can set a default of `0` here, since most people only have one camera available.